src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -93,6 +93,20 @@ passes `-L`, a flag that helps rustdoc find the dependencies
 your code relies on. If our project used dependencies, we'd get
 documentation for them as well!
 
+## Crate level documentation
+From the above examples, you can see that using the `///` syntax
+documents the next item. But that can't be used when you are
+documenting the entire crate, because there is nothing before the crate.
+It is where everything is placed under. So, in order to document an entire
+crate, you use this `//!` syntax. For example
+
+``` rust
+//! This is my first rust crate
+```
+
+For more information about the `//!` syntax, see [the Book](https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#commenting-contained-items).
```

Comment:
nit: I think this would be better on two lines so it's not as long.
```suggestion
For more information about the `//!` syntax, see [the Book].

[the Book]: https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#commenting-contained-items
```

Reply:
So what should I do,? Which one should I resolve

Reply:
So what should I do,? Which one should I resolve

Reply:
You can make multiple changes at once, you don't have to make them one at a time and ask between each :)